presto-docs/src/main/sphinx/connector/localfile.rst

```diff
@@ -0,0 +1,45 @@
+====================
+Local File Connector
+====================
+
+The local file connector allows querying data stored on the local
+filesystem of each worker.
+
+Configuration
+-------------
+
+To configure the local file connector, create a catalog properties file
+``etc/catalog/localfile.properties`` with the following contents:
```

Comment:
This is the language we use elsewhere. It is simpler to tell people exactly what to do, then explain in detail elsewhere for advanced users.